index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello, World!</h1>
```

Comment:
Please add more text to test your understanding of comments.

Reply:
Added more text to make clear, that I understood coment